### ISSUE TITLE
main.js is missing. can't run the makefile

### ISSUE THREAD

**Issue by prubel** [asciinema/vt-clj#8]
I was hoping to run the test but when I try to run make it's failing: 
  $ make
  ...
  Error: Cannot find module '.../vt/main.js'
  ...
is there a file missing somewhere? A note on how to get things running would be helpful.

**Comment by prubel** [comment]
responding to myself here. main.js was generated when I ran 
  $ lein cljsbuild once
I also needed to modify project.clj due to this failure:
  ...
  Caused by: clojure.lang.ExceptionInfo: No such namespace: doo.runner, could not locate doo/runner.cljs, doo/runner.cljc, or JavaScript source providing "doo.runner" in file test/asciinema/test_runner.cljs {:tag :cljs/analysis-error}
  ...

I'm new to clojurescript so I certainly don't know if this is the right way to do things. I can make a pull request but it might take a bit. 
Here's a diff in the meantime

diff --git a/project.clj b/project.clj
index 540ef2a..58ba502 100644
--- a/project.clj
+++ b/project.clj
@@ -22,6 +22,7 @@
 
   :profiles {:dev {:dependencies [[com.cemerick/piggieback "0.2.1"]
                                   [org.clojure/tools.nrepl "0.2.10"]
+                                  [lein-doo "0.1.6"]
                                   [environ "1.0.1"]]
                    :plugins [[refactor-nrepl "1.1.0"]]}
              :repl {:plugins [[cider/cider-nrepl "0.10.0"]]}}
